packages/core-bridge/src/helpers.rs

```diff
@@ -226,23 +226,64 @@ pub fn hashmap_to_js_value<'a>(
 fn snake_to_camel(input: String) -> String {
     match input.find('_') {
         None => input,
-        Some(first) => {
+        Some(_) => {
             let mut result = String::with_capacity(input.len());
+            snake_to_camel_mut(&input, &mut result);
+            result
+        }
+    }
+}
+
+// Convert from Core's log "targets" (actually full module name, like "temporal_sdk_core::worker::workflow"),
+// to the format used in our JavaScript side logger (i.e. "core/worker/workflow").
+pub fn rust_package_to_js_style(input: &str) -> String {
```

Comment:
This seems slightly overkill to me, but you've done it so no reason to get rid of it.

Reply:
+1, just slap a `core` tag on every log record and be done with it.
Leave the Rust style tags verbatim so they're consistent when logging to console or being forwarded to lang.

Reply:
I'd strongly prefer consistency of the TS SDK with itself than TS SDK with other SDKs. We already took the decision of converting attribute names to TS style in the past, for that reason, and I don't think we should diverge here.

Reply:
However, I wouldn't mind making TS SDK itself adopt the rust syntax for that specific purpose. Does that make sense to you?